petastorm/reader.py

```diff
@@ -123,14 +124,16 @@ def make_reader(dataset_url,
         here: https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetDataset.html
     :param s3_config_kwargs: dict of parameters passed to ``botocore.client.Config``
     :param zmq_copy_buffers: A bool indicating whether to use 0mq copy buffers with ProcessPool.
+    :param filesystem: A filesystem to use. Will ignore s3_config_kwargs and other filesystem configs if it's provided.
```

Comment:
Let's mention that this is a pyarrow filesystem is expected.

Reply:
Sure.